plural/terraform/aws/terraform.tfvars

```diff
@@ -5,4 +5,9 @@ plural_assets_bucket = {{ .Values.assets_bucket | quote }}
 plural_images_bucket = {{ .Values.images_bucket | quote }}
 plural_namespace = {{ .Namespace | quote }}
 cluster_name = {{ .Cluster | quote }}
-shell_subnet_ids = {{ $bootstrap.cluster_worker_private_subnet_ids | toJson }}
\ No newline at end of file
+shell_subnet_ids = {{ $bootstrap.cluster_worker_private_subnet_ids | toJson }}
+node_role_arn = {{ (index $bootstrap.node_groups 0).node_role_arn | quote }}
```

Comment:
how does the shell nodes get this?  It's probably reusable

Reply:
Inzong think it does. It’s not passing a value for the arn in any case.

Reply:
Which might be good from a security standpoint.

Reply:
looks like it's done within a module library submodule here: https://github.com/pluralsh/module-library/blob/main/terraform/eks-node-groups/multi-az-node-group/main.tf#L9